src/SourceBuild/tarball/content/repos/Directory.Build.targets

```diff
@@ -496,17 +496,11 @@
 
     <ZipFileExtractToDirectory Condition="'@(_BuiltIntermediatePackages)' != ''"
                             SourceArchive="%(_BuiltIntermediatePackages.Identity)"
-                            DestinationDirectory="$(SourceBuiltPackagesPath)extractArtifacts/"
+                            DestinationDirectory="$(SourceBuiltPackagesPath)extractArtifacts/%(_BuiltIntermediatePackages.FileName)/"
                             OverwriteDestination="true" />
 
     <ItemGroup Condition="'@(_BuiltIntermediatePackages)' != ''">
-      <SourceBuiltNupkgFiles Include="$(SourceBuiltPackagesPath)extractArtifacts/**/*.nupkg" />
```

Comment:
Could you also remove the nupkgs in src/SourceBuild/tarball/content/packages please?

Reply:
Sorry, looks like those were removed already and we missed this.

Reply:
I think they're already removed.  Are you talking about the ones that were checked in?